### PR TITLE
7953 Geo-location of Pricing Page

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-7953

### CODE REVIEW

File: source/javascripts/partials/_pricing.es6 [hunk 1]
```diff
@@ -248,7 +251,54 @@ const setupFaqBlock = () => {
...
+  console.log('Updating for usd');
+  state.currency = 'dollar';
+}
+
+const handleGeolocation = (text) => {
+  var data = {};
+  try {
+    data = JSON.parse(text);
+    if (data.country_code && eumembers[data.country_code]) {
+      updatePageForGBR;
```
Comment (AndreyNikitin) [inline source/javascripts/partials/_pricing.es6]: should be function

Comment (denpira) [inline source/javascripts/partials/_pricing.es6]: Done

File: source/javascripts/partials/_pricing.es6 [hunk 2]
```diff
@@ -248,7 +251,54 @@ const setupFaqBlock = () => {
...
+    updateDisplay();
+  } catch (e) {
+    console.log('Error on parsing geolocation data');
+    data = {};
+  }
+}
+
 $(document).ready(function () {
+  var geoplugin = '//freegeoip.net/json/';
+  loadXMLDoc(geoplugin, handleGeolocation);
```
Comment (AndreyNikitin) [inline source/javascripts/partials/_pricing.es6]: what if plugin won't load? is there default state?

Comment (denpira) [inline source/javascripts/partials/_pricing.es6]: Done

File: source/javascripts/partials/_pricing.es6 [hunk 3]
```diff
@@ -248,7 +258,55 @@ const setupFaqBlock = () => {
...
+  console.log('Updating for usd');
+  state.currency = 'dollar';
+}
+
+const handleGeolocation = (text) => {
+  var data = {};
+  try {
+    data = JSON.parse(text);
+    if (data.country_code && eumembers[data.country_code]) {
+      updatePageForGBR();
```
Comment (AndreyNikitin) [inline source/javascripts/partials/_pricing.es6]: gbr only for UK, do not nedd for eu members, think ask about euro price for them.

File: source/pricing.html.erb [hunk 4]
```diff
@@ -31,6 +31,7 @@ fixed: false
   </ul>
   
   <div class="tab-content pricing-tab pricing-tab-content">
+    <!--
```
Comment (MaximKraev) [inline source/pricing.html.erb]: remove unused code, we have it in git history

Comment (denpira) [inline source/pricing.html.erb]: Done

**Comment by MaximKraev** [comment]
👍

**Pull request merged**